.github/workflows/main.yaml

```
@@ -124,10 +124,16 @@ jobs:
         run: sudo apt install libgtk-3-dev libssh2-1-dev libglib2.0-dev libgraphene-1.0-dev libcairo-gobject2 libcairo2-dev
         if: matrix.os == 'ubuntu-20.04'
       - name: Install toolchain packages with pacman
-        run: pacman --noconfirm -S base-devel mingw-w64-${{ matrix.arch }}-toolchain
+        run: curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz
+        run: curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz.sig
+        run: pacman -U --noconfirm --config <(echo) msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz
+        run: pacman --noconfirm -Syu base-devel mingw-w64-${{ matrix.arch }}-toolchain
```

```suggestion
          run: curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz
        - run: curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz.sig
        - run: pacman -U --noconfirm --config <(echo) msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz
        - run: pacman --noconfirm -Syu base-devel mingw-w64-${{ matrix.arch }}-toolchain
```
The first run block has a name